src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -56,10 +57,29 @@ User loggedInUser = UserStore.getInstance().getUser(user);
 
   <div id="container">
 
+    <% if(request.getAttribute("error") != null){ %>
+        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+    <% } %>
+
     <h1><%= conversation.getTitle() %>
       <a href="" style="float: right">&#8635;</a></h1>
-
+    
+    <% if (loggedInUser != null) { %>
+        <label class="form-control-label">Current tags:</label>
+        <p><%= request.getAttribute("tags") %><p>
```

Comment:
Iterate through List of Tags to make each of them a link to their "Tag profile page"